DataFormats/ParticleFlowReco/interface/PFRecHit.h

```diff
@@ -161,70 +148,40 @@ namespace reco {
     /// comparison < operator
     bool operator< (const PFRecHit& rhs) const { return (energy_< rhs.energy_); }
 
-    friend std::ostream& operator<<(std::ostream& out, 
-                                    const reco::PFRecHit& hit);
-
-    const edm::RefToBase<CaloRecHit>& originalRecHit() const {
-      return originalRecHit_;
-    }
-
-    template<typename T> 
-    void setOriginalRecHit(const T& rh) {
-      originalRecHit_ = edm::RefToBase<CaloRecHit>(rh);
-    }
-
+ 
   private:
 
-    // original rechit
-    edm::RefToBase<CaloRecHit> originalRecHit_;
-
-    ///C cell detid - should be detid or index in collection ?
-    unsigned            detId_;             
+    Neighbours buildNeighbours(unsigned int n) const { return  Neighbours(&neighbours_.front(),n);}
+    
+    /// cell geometry
+    CaloCellGeometry const * caloCell_=nullptr;
```

Comment:
Because that is what ROOT I/O does when it encounters a pointer, it tries to write the object pointed to out to the file.